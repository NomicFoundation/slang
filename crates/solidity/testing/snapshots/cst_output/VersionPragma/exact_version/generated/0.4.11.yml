# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Source: >
  1  │ solidity 0.8.0                                                                   │ 0..14

Errors: []

Tree:
  - VersionPragma (Rule): # 0..14 "solidity 0.8.0"
      - Solidity (Token): "solidity" # 0..8
      - Group: # 9..14 "0.8.0"
          - VersionPragmaSpecifier (Rule): # 9..14 "0.8.0"
              - Group: # 9..14 "0.8.0"
                  - VersionPragmaValue (Token): "0" # 9..10
                  - Period (Token): "." # 10..11
                  - VersionPragmaValue (Token): "8" # 11..12
                  - Period (Token): "." # 12..13
                  - VersionPragmaValue (Token): "0" # 13..14
