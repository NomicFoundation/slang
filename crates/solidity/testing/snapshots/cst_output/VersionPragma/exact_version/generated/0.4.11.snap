# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Errors: 0

█ Rule(VersionPragma) - (0..14)                             │
├──► Token(Solidity) - (0..8)                               │ solidity
└──█ Rule(VersionPragmaSpecifiers) - (9..14)                │
   └──█ Rule(VersionPragmaSpecifier) - (9..14)              │
      └──█ Rule(SeparatedVersionPragmaValues) - (9..14)     │
         ├──► Token(VersionPragmaValue) - (9..10)           │ 0
         ├──► Token(Period) - (10..11)                      │ .
         ├──► Token(VersionPragmaValue) - (11..12)          │ 8
         ├──► Token(Period) - (12..13)                      │ .
         └──► Token(VersionPragmaValue) - (13..14)          │ 0
